tests/src/python/CMakeLists.txt

```diff
@@ -386,22 +388,35 @@ if (ENABLE_PGTEST)
   ADD_PYTHON_TEST(PyQgsProviderConnectionPostgres test_qgsproviderconnection_postgres.py)
   if (WITH_SERVER)
     ADD_PYTHON_TEST(PyQgsServerWMSGetFeatureInfoPG test_qgsserver_wms_getfeatureinfo_postgres.py)
+    # project contains connection to Postgres database
+    ADD_PYTHON_TEST(PyQgsServerWMSGetFeatureInfo test_qgsserver_wms_getfeatureinfo.py)
```

Comment:
Can you move the tests that use PG from PyQgsServerWMSGetFeatureInfoto PyQgsServerWMSGetFeatureInfoPG ?

Reply:
Done